ember-client/shared/src/main/scala/org/http4s/ember/client/EmberClientBuilder.scala

```diff
@@ -429,8 +429,8 @@ object EmberClientBuilder extends EmberClientBuilderCompanionPlatform {
     val acgFixedThreadPoolSize: Int = 100
     val chunkSize: Int = 32 * 1024
     val maxResponseHeaderSize: Int = 4096
-    val idleConnectionTime: FiniteDuration = org.http4s.client.defaults.RequestTimeout
-    val timeout: Duration = org.http4s.client.defaults.RequestTimeout
+    val idleConnectionTime: FiniteDuration = org.http4s.ember.core.Defaults.idleTimeout
+    val timeout: Duration = org.http4s.ember.core.Defaults.idleTimeout
```

Comment:
Hmm, maybe changing the default `timeout` is not desired. I was thinking that `45s` is quite a lot but `60s` 🤯

Reply:
My bad 😬 . I reverted the timeout from previous value.